Source/Scene/ModelExperimental/InstancingPipelineStage.js

```diff
@@ -110,7 +112,14 @@ InstancingPipelineStage.process = function (renderResources, node, frameState) {
     }
   }
 
-  renderResources.instanceCount = node.instances.attributes[0].count;
+  processFeatureIdAttributes(
```

Comment:
Yes - this can be tested in this [Sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=bVRhb9MwEP0rpl9ItM5JWdlgLRNQtlHRwUQr+EAQcpNLa+HYke10LWj/nbOTtGup1Crx+b3Lu7tnr5gmKw4PoMkbIuGBjMDwqqDffCxIOqlfj5S0jEvQSSccJDKRK+SVynDLlURmwxoxbfGNyTOaa1V8gIUGMEEiCTntvTij8UW/f9573XWBfp/GL+Ozi/jcL3txnEiX22VelgdyPgLLuFzcc5suvyohgrhL3G8nJudryG40K2CmmTS50sVO1zZkqFApEzVO3Ww5t4C1Mau015p0pNJ2mXS69eoBjE06tbz6a4XKQGD+unXUpMinpeYFNmQFhrIs85ma7985+PW6BEQAdlL49twKmwd/HYyQBb5f4qeiaQmpiT4wyyLPMpGDTRTLQEfv1XosjWUyhSxaiNlNNFfrU96GqMvSqCa1xjtmNV9fHmnE8qClT9sR1BnIdsTdNoCT2b43Oa+F4KVRPKPfb6ev+tvtIxOpt8JWIJe8qIp7xIkp/wOXpPfiVbOXA7OVhnH2zmIB88rCWGaAhcRu/zFsh+GLpBpr2dxjS7kBapcgg7ySqfdm4BEh8X1uxpWiJs1oLjYz9V5V0vVhWi5BQ42m871glzRDyir0CCZFFfRlrQNFPO5sYUBAarHqWn0dby2CR0FOS5bC9Qo98JHJTLg42LEsK/vOy93pVnICuR0Jnv720yjUCpx3sPCmFmf6+jv/GdGRWgJtZ+hkEsJzEjwLWmJrHZXvWbXRH4Zt5RpwLX2CR38m8H9QLIrIt2WTWl+pFXrebj5jw80OQBdg759uBeGOI0Au7BLBe2Rahz0MnUQCB+WIigf4GDasATk54a3oQwGHKX/wn4PGq6mSRgnAy2ER7FFOyPNL8hwfR5TvIUNfwbY3TxMe9Mlbptu2e3pgi9mmBDq5vpn9Gk3Go0/hoNPtDI3dCLhqz9VbXpR4PZFKi4DSyEJRCobXbjSv0t9gaWpM004yjJ5ShxlfEZ69OXKpk1QwY3Anr4Q/i0nnahgh/j+qUP7a+LICLdjGwZa9q0kdpJQOI1weZ1qlxJzpg8z/AA).